website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -55,6 +63,10 @@ The following attributes are exported:
 
 * `kubernetes_version` - The version of Kubernetes used on the managed Kubernetes Cluster.
 
+* `private_link_enabled` - If private IP address for k8s API is enabled through private link.
```

Comment:
since this is a boolean value - could we rephrase this slightly:

```suggestion
* `private_link_enabled` - Does this Kubernetes Cluster have the Kubernetes API exposed via Private Link?
```

Reply:
Resolved in new commit